<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Portal</class>
 <widget class="QMainWindow" name="Portal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XShot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="AfPicture" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>截图</string>
    </property>
    <addaction name="actionDesktop"/>
    <addaction name="separator"/>
    <addaction name="actionWindow"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>保存</string>
    </property>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDesktop"/>
   <addaction name="separator"/>
   <addaction name="actionWindow"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCapture">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>截图</string>
   </property>
   <property name="toolTip">
    <string>截图</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存图片</string>
   </property>
   <property name="toolTip">
    <string>保存图片</string>
   </property>
  </action>
  <action name="actionDesktop">
   <property name="text">
    <string>截取桌面</string>
   </property>
   <property name="toolTip">
    <string>截取桌面</string>
   </property>
  </action>
  <action name="actionWindow">
   <property name="text">
    <string>截取窗口</string>
   </property>
   <property name="toolTip">
    <string>截取窗口</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于本程序</string>
   </property>
   <property name="toolTip">
    <string>关于本程序</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>关闭程序</string>
   </property>
   <property name="toolTip">
    <string>关闭程序</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AfPicture</class>
   <extends>QFrame</extends>
   <header location="global">afqt/AfPicture.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
